test/verify/check-connection

```diff
@@ -965,6 +965,51 @@ until pgrep -f '^/usr/bin/cockpit-bridge.*--privileged'; do sleep 1; done
         m.write("/etc/cockpit/ws-certs.d/0-self-signed-ca.pem", "FAKE CERT FOR TESTING\n")
         self.assertEqual(m.execute("curl -sfS http://localhost:9090/ca.cer"), "FAKE CERT FOR TESTING\n")
 
+    def testAuthUnixPath(self):
+        '''test UnixPath for auth method in cockpit.conf'''
+        m = self.machine
+
+        # selinux doesn't like this...
+        if m.image in ['centos-8-stream', 'fedora-34', 'fedora-35', 'fedora-coreos', 'rhel-8-5', 'rhel-8-6', 'rhel-9-0']:
```

Comment:
I propose to check `selinuxenabled` on the image, which will be less error prone.

Reply:
I was looking for something like this!  Thanks!

Reply:
I thought this was an attribute on the image.... This seems annoying to use.  What would you propose exactly?

```
    if m.execute("selinuxenabled && echo 'enabled' && setenforce 0 || true"):
         self.allow_journal_messages()...
```
?

Reply:
Yes, something like this. The existing cases (`git grep selinuxenabled test` all do an `if selinuxenabled; then action..` kind of thing. (Please use `if`, it feels a little cleaner than alternative truths; but no strong opinion)

Also, does this actually work anywhere? I have a déjà vu that I worked on some patch to our SELinux policy to fix that, but that's not in this series? I.e. it feels like we should pull that into this series and drop all these special cases?

Reply:
The problem is that I want to conditionalise ignoring this journal message.  Or do you think I should just do that unconditionally?

Reply:
I am mostly concerned about how we get rid of this.

 * Is this something we can fix in our own SELinux policy? If so, then let's do that right away (I'm happy to help with this, of course!), and we shouldn't need the test quirk at all.
 *  If this is outside of our own policy, then we need to have a plan how to enable this: A bugzilla against selinux-policy, reference it in a comment, and *then* the OS list actually makes sense -- we can then drop fixed OSes to make sure it works and doesn't regress.

So all in all, I retract my proposal to use `selinuxenabled`, it doesn't make sense.

Reply:
Take a look at 
https://github.com/cockpit-project/cockpit/commit/98e1ef7f6dd238239380ef9bbf66f778f952d590 for what the fix would look like.  I didn't include it here because I'm not sure we should carry that just for a test case...

Reply:
Ah, I remember this :wink: I wouldn't mind adding that selinux fix right now to spare us the cleanup in the future. But this confirms that the fix will not be OS specific, so the long and detailed OS list does not make much sense after all, and checking `selinuxenabled` seems right. Sorry for having forgotten all the context here in the last three weeks!

So if you prefer to withhold that policy, let's check selinuxenabled and otherwise keep the current quirk.

Reply:
I think maybe we could try to merge everything from the other PR except the two last commits that actually do the switchover.  That would provide a semi-supported mechanism to use this stuff without it being "the way".  In that case I'd pick the selinux fix.